src/sage/modular/arithgroup/congroup_gamma1.py

```diff
@@ -650,3 +657,246 @@ def dimension_new_cusp_forms(self, k=2, eps=None, p=0, algorithm="CohenOesterle"
         eps_p = eps.restrict(N//p)
         old = Gamma1_constructor(N//p).dimension_cusp_forms(k, eps_p, algorithm)
         return self.dimension_cusp_forms(k, eps, algorithm) - 2*old
+
+
+
```

Comment:
```suggestion
        return self.dimension_cusp_forms(k, eps, algorithm) - 2*old


```
One too many blank lines.